src/reduxTypes.js

```diff
@@ -247,7 +261,7 @@ export type RealmState = {|
   nonActiveUsers: User[],
   filters: RealmFilter[],
   emoji: RealmEmojiById,
-  jitsiServerUrl: string | null,
+  videoChatProvider: VideoChatProvider | null,
```

Comment:
I like this way of representing the data!

Reply:
Excellent! Yeah, learning about disjoint unions in Flow and discriminated unions in Typescript was a life-changing moment for me.